packages/playwright-test/src/loader.ts

```diff
@@ -678,3 +680,19 @@ export function folderIsModule(folder: string): boolean {
   // Rely on `require` internal caching logic.
   return require(packageJsonPath).type === 'module';
 }
+
+function fixWin32FilepathCapitalization(file: string): string {
+  /**
+   * On Windows with PowerShell <= 6 it is possible to have a CWD with different
+   * casing than what the actual directory on the filesystem is. This can causes
+   * that we require the file multiple times with different casing. To mitigate
+   * this we get the actual underlying filesystem path and use that.
+   * https://github.com/microsoft/playwright/issues/9193#issuecomment-1219362150
+   */
+  const realFile = fs.realpathSync.native(file);
```

Comment:
Does it have to be sync? It affects every _requireOrImport so might be noticeable on windows.